neptune/new/internal/backends/utils.py

```diff
@@ -130,6 +130,9 @@ def create_swagger_client(url: str, http_client: HttpClient) -> SwaggerClient:
 
 
 def verify_client_version(client_config: ClientConfig, version: Version):
+    if str(version) == '0+unknown':
```

Comment:
Add some comment when version is `'0+unknown'` and that it should be the case only for e2e tests.

You've updated e2e tests to test recent master instead of recent version published on PIP, right?

Reply:
Fixed in https://github.com/neptune-ai/neptune-client/pull/678